src/share/poudriere/jail.sh

```diff
@@ -63,14 +63,14 @@ Options:
                      obtaining and building the jail. See poudriere(8) for more
                      details. Can be one of:
                        'allbsd', 'ftp-archive', 'ftp', 'freebsdci', 'http',
-		       'null', 'src=PATH', 'tar=PATH', 'url=URL', or
+		       'null', 'src=PATH', 'tar=PATH', 'url=URL', 'pkgbase=repo' or
 		       '{git,svn}{,+http,+https,+file,+ssh}' (e.g., 'git+https').
                      The default is '${METHOD_DEF}'.
     -P patch      -- Specify a patch to apply to the source before building.
     -S srcpath    -- Specify a path to the source tree to be used.
     -D            -- Do a full git clone without --depth (default: --depth=1)
     -t version    -- Version of FreeBSD to upgrade the jail to.
-    -U url        -- Specify a url to fetch the sources (with method git and/or svn).
+    -U url        -- Specify a url to fetch the sources (with method git, svn and pkgbase).
```

Comment:
I realise this is unrelated to this Pull Request, but does it still make sense to keep supporting?